CHANGELOG.rst

```diff
@@ -17,11 +17,16 @@
   is specified on the command line together with the ``--pyargs``
   option.  Thanks to `@taschini`_ for the PR (`#1597`_).
 
+* Rename ``getfuncargvalue`` to ``getfixturevalue``. `getfuncargvalue`
```

Comment:
Should probably also use double-backticks with the second `getfuncargvalue` as well?